doc/asa.rst

```diff
@@ -3,6 +3,22 @@
 Security Advisory
 #################
 
+Addressed in ACRN v1.6.1
+************************
+
+We recommend that all developers upgrade to this v1.6.1 release (or later), which
+addresses the following security issue that were discovered in previous releases:
```

Comment:
Change ``were`` to ``was``

Reply:
to expedite things, I made this change directly to @yonghuah branch since it was a simple grammar fix